### PR TITLE
chore(redis): migrate redis examples

### PR DESCRIPTION
**Pull request by haddasbronfman**
## Which problem is this PR solving?

https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1155

## Short description of the changes

migrate redis exampled from 'examples' folder to instrumentation-redis folder

## Checklist

- [ ] Ran `npm run test-all-versions` for the edited package(s) on the latest commit if applicable.

### CODE REVIEW

File: plugins/node/opentelemetry-instrumentation-redis/examples/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "@opentelemetry/sdk-trace-base": "^1.0.0",
+    "axios": "^0.21.1",
+    "express": "^4.17.1",
+    "redis": "^2.8.0"
+  },
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js#readme",
+  "devDependencies": {
+    "cross-env": "^6.0.0",
+    "ts-node": "^10.6.0",
+    "typescript": "^4.6.2"
```
Comment (osherv) [inline plugins/node/opentelemetry-instrumentation-redis/examples/package.json:50]: You should lock down the typescript version to `4.3.5`

Comment (osherv) [reply plugins/node/opentelemetry-instrumentation-redis/examples/package.json:50]: That's why your CI tests fails

Comment (haddasbronfman) [reply plugins/node/opentelemetry-instrumentation-redis/examples/package.json]: thanks! I'll fix that

File: plugins/node/opentelemetry-instrumentation-redis/examples/src/client.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+// eslint-disable-next-line import/order
```
Comment (osherv) [inline plugins/node/opentelemetry-instrumentation-redis/examples/src/client.ts:3]: You can remove those

File: plugins/node/opentelemetry-instrumentation-redis/examples/src/client.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import { setupTracing } from "./tracer";
```
Comment (osherv) [inline plugins/node/opentelemetry-instrumentation-redis/examples/src/client.ts:4]: Run a linter to fix the conventions

File: plugins/node/opentelemetry-instrumentation-redis/examples/src/server.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+// eslint-disable-next-line import/order
```
Comment (osherv) [inline plugins/node/opentelemetry-instrumentation-redis/examples/src/server.ts:3]: Can be removed

**Review by blumamir — APPROVED** [review]
LGTM. 
Wrote a few non-blocking suggestions

File: plugins/node/opentelemetry-instrumentation-redis/examples/src/server.ts [hunk 5]
```diff
@@ -19,7 +19,7 @@ const PORT = 8080;
 async function setupRoutes() {
   const redis = await redisPromise;
 
-  app.get('/run_test', async (req, res) => {
+  app.get('/run_test', async (req: any, res: any) => {
```
[the remark this answers is not in the document]
Comment (haddasbronfman) [reply plugins/node/opentelemetry-instrumentation-redis/examples/src/server.ts:22]: I'll fix

File: plugins/node/opentelemetry-instrumentation-redis/examples/package.json [hunk 6]
```diff
@@ -30,20 +31,22 @@
...
+    "@opentelemetry/api": "^1.0.0",
+    "@opentelemetry/exporter-jaeger": "^1.0.0",
+    "@opentelemetry/exporter-zipkin": "^1.0.0",
+    "@opentelemetry/instrumentation": "^0.32.0",
+    "@opentelemetry/instrumentation-http": "^0.32.0",
+    "@opentelemetry/instrumentation-redis": "^0.32.0",
+    "@opentelemetry/sdk-trace-node": "^1.0.0",
+    "@opentelemetry/sdk-trace-base": "^1.0.0",
     "axios": "^0.21.1",
     "express": "^4.17.1",
```
[the remark this answers is not in the document]
Comment (haddasbronfman) [reply plugins/node/opentelemetry-instrumentation-redis/examples/package.json:43]: good point

File: plugins/node/opentelemetry-instrumentation-redis/examples/package.json [hunk 7]
```diff
@@ -7,10 +7,11 @@
   "scripts": {
     "docker:start": "docker run -d -p 6379:6379 --name otjsredis redis:alpine",
     "docker:stop": "docker stop otjsredis && docker rm otjsredis",
```
[the remark this answers is not in the document]
Comment (haddasbronfman) [reply plugins/node/opentelemetry-instrumentation-redis/examples/package.json:9]: I'll change that. thanks.

**Pull request merged**